transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/CctTransportBackend.java

```diff
@@ -54,13 +55,11 @@
 import java.util.List;
 import java.util.Map;
 import java.util.TimeZone;
-import java.util.logging.Level;
-import java.util.logging.Logger;
 import java.util.zip.GZIPOutputStream;
 
 final class CctTransportBackend implements TransportBackend {
 
-  private static final Logger LOGGER = Logger.getLogger(CctTransportBackend.class.getName());
+  private static final String LOG_TAG = "CctTransportBackend";
```

Comment:
Any reason for not preferring the class.getname()?

Reply:
yes, `class.getName()` will return the name of the class after proguarding, so it could end up being "a", so to avoid it I explicitly preserve the class name as a constant string

Reply:
Makes sense